### ISSUE TITLE
Credit Card Charge Response getBody() method returns an empty value

### ISSUE THREAD

**Issue by TM2020IT** [intuit/PHP-Payments-SDK#30]
This wasn't an issue until today, no other website or code changes were made in the last week.  My only guess is that intuit changed their API response handling.

Within the PaymentClient class the "charge" method has a duplicate httpClient->send method call. This is now causing the first call to have a valid "$response->getBody()" value, while the second call, which is the one that gets returned by the "charge" method to have an empty body value. Up until today the second "send" method call returned a valid body response. Today's issue caused the credit card to be charged successfully, but the order to fail because the body was empty and the response couldn't be verified as successful or not. Commenting out line 95 resolves the issue.

Line 95: $response = $this->httpClient->send($request);

Another item to note, there is also a duplicate "send" method call on line 110 within the voidChargeTransaction method of this same class.

**Comment by Nebual** [comment]
Seconded, we just started getting reports today of payments failing, and this seems to be the root.

I'd guess their API just implemented some sort of idempotency check _today_, ignoring duplicate requests (with the same request-id), and unfortunately this SDK specifically performs exactly that.

**Comment by sarmadmakhdoom** [comment]
Yeah we are also facing the same issue with charge from Friday. Can you please share any solution?

**Comment by sarmadmakhdoom** [comment]
<img width="745" alt="Screen Shot 2021-11-22 at 12 02 16 AM" src="https://user-images.githubusercontent.com/3525711/142775452-630958eb-5873-45cd-bdae-4cb98a0e4f8e.png">

We see in the SDK source code, sending the same request twice in charge method. I am not sure why this is the case but removing the first send solves the problem. But why there were two of them in the first place. Are we missing anything here?

**Comment by Nebual** [comment]
Yeah that double line seems to be the cause of the problem. If you need a composer package with that fix, add this to your "repositories" in `composer.json`:
```
  "repositories": [
    {
      "type": "vcs",
      "url": "https://github.com/Nebual/PHP-Payments-SDK.git"
    },
  ]
```
And then run:
```
composer require quickbooks/payments-sdk="dev-master#f6af73f85e412b7e87b8170c69c2647056641dfa"
```

That'll be release official 1.0.8 + #31 's fix
